CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 All notable changes to this project will be documented in this file.
 
 ##  Unreleased
+- Added: Added "cso_schedule" to variable
```

Comment:
presume you don't need to "to variable" just " variable"

Reply:
Can probably drop the double "Added: Added" so just "Added: "cso_schedule" to variable"